### PR TITLE
Setup new type datetime_immutable_no_tz for ODM

### PR DESCRIPTION
**Pull request by curzio-della-santa**

### CODE REVIEW

File: src/Detail/Persistence/Doctrine/ODM/Types/DatetimeNoTzType.php [hunk 1]
```diff
@@ -80,4 +80,6 @@ public static function craftDateTime($seconds, $microseconds = 0)
     {
         return \DateTime::createFromFormat('U', $seconds, new \DateTimeZone('UTC'));
     }
+
+
```
Comment (ivan-wolf) [inline src/Detail/Persistence/Doctrine/ODM/Types/DatetimeNoTzType.php]: Remove the added empty lines.

File: tests/DetailTest/Persistence/Options/ModuleOptionsTest.php [hunk 2]
```diff
@@ -43,4 +43,16 @@ public function testDoctrineDatetimeNoTzTypeCanBeSet()
 
         $this->assertTrue($this->options->getDoctrine()->registerDatetimeNoTzType());
     }
+
+    public function testDoctrineDatetimeImmutableNoTzTypeCanBeSet()
```
Comment (ivan-wolf) [inline tests/DetailTest/Persistence/Options/ModuleOptionsTest.php]: Wrong place, belongs to `DoctrineOptionsTest`.

Comment (curzio-della-santa) [inline tests/DetailTest/Persistence/Options/ModuleOptionsTest.php]: All the tests in this class are doctrine options tests, I should refactor the Class and file name

File: tests/DetailTest/Persistence/Options/DoctrineOptionsTest.php [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 namespace DetailTest\Persistence\Options;
 
-class ModuleOptionsTest extends OptionsTestCase
+class DoctrineOptionsTest extends OptionsTestCase
```
Comment (ivan-wolf) [inline tests/DetailTest/Persistence/Options/DoctrineOptionsTest.php:7]: This is completely wrong now...

**Pull request merged**